buildconfig/Jenkins/Conda/build

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -ex
+
+# This script expects a setup environment with all dependencies availiable in
+# the environment
+#
+# Expected args:
+#   1. WORKSPACE: path to the workspace/source code that this should run inside
+#   2. CMAKE_PRESET: the CMake preset that should be ran to generate the cmake files for this CI job
+#   3. ENABLE_DOCS: build the user docs
+#   4. ENABLE_DEV_DOCS: build the user docs
+#   5. ENABLE_BUILD_CODE: whether or not to build the main code target
+#   6. ENABLE_UNIT_TESTS: whether or not to build the unit tests target
+#   7. ENABLE_SYSTEM_TESTS: whether or not to build the system test data
+#   8. EXTRA_CMAKE_FLAGS: Extra flags to pass directly to cmake, enclose in "".
+#   9. BUILD_THREADS: Pass the number of threads that can be used to build with
+
+WORKSPACE=$1
+CMAKE_PRESET=$2
+ENABLE_DOCS=$3
+ENABLE_DEV_DOCS=$4
+ENABLE_BUILD_CODE=$5
+ENABLE_UNIT_TESTS=$6
+ENABLE_SYSTEM_TESTS=$7
+EXTRA_CMAKE_FLAGS=$8
+BUILD_THREADS=$9
+
+# Only pass MANTID_DATA_STORE to CMake if present in the environment otherwise rely on CMake default
+if [![ -n "${MANTID_DATA_STORE}" ]]; then
```

Comment:
This can be simplified to `-z "${MANTID_DATA_STORE}"` since were already wedded to bash

Reply:
```suggestion
if [[ -n "${MANTID_DATA_STORE}" ]]; then
```